scripts/bio-rdf-validator.imjoy.html

```diff
@@ -0,0 +1,43 @@
+<config lang="json">
+{
+  "name": "BIO-RDF-Validator",
+  "type": "web-python",
+  "version": "0.1.0",
+  "description": "A spec validator for bioimage RDF files.",
+  "tags": [],
+  "ui": "",
+  "cover": "",
+  "inputs": null,
+  "outputs": null,
+  "flags": [],
+  "icon": "extension",
+  "api_version": "0.1.8",
+  "env": "",
+  "permissions": [],
+  "requirements": ["bioimageio.spec==0.4.3"],
```

Comment:
We need to change here when we have a new version for the uploader.

Reply:
Can we somehow read this from https://github.com/bioimage-io/spec-bioimage-io/blob/main/bioimageio/spec/VERSION instead?

Reply:
How can that be different with not pin the version? I think the point to pin is that we can bump it manually and not break the website even when the latest version does work as expected.

Reply:
ah ok, I see. Yes then let's keep the pin here and bump it manually when needed.